docs/run-taco.md

```diff
@@ -6,11 +6,19 @@ Note: the below is intended for connector developers. Customer-facing documentat
 
 # Load order and class name collisions
 
+## Tableau 2022.2 and above
```

Comment:
Can you move this part to the 2022.2 branch

Reply:
Reverted. Will move to 2022.2 branch after this change is merged to master and then to the 2022.2 branch to try and avoid merge conflicts.